worker/task.go

```diff
@@ -61,6 +62,17 @@ func (t *Task) IsResolved() bool {
 	return t.resolved
 }
 
+// HexDump returns a byte dump of the task, in the same format as `hexdump -C`.
+// This is useful for debugging/logging purposes.
+func (t *Task) HexDump() string {
+	return hex.Dump(t.Bytes())
+}
+
+// String returns a dump of the task contents as a string.
```

Comment:
Yep, :+1: When my brain is in code-writing mode it's hard to switch over to English-writing mode.